linux/CGroupMem.c

```
@@ -0,0 +1,236 @@
+            }
+         }
+      }
+
+      if (!eol)
+         break;
+      p = eol + 1;
+   }
+   return false;
+}
+
+bool CGroupMem_parseUsage(const char* content, unsigned long long* outKB) {
```

Cppcheck (2.21.0)

[style] The function 'CGroupMem_parseUsage' should have static linkage since it is not used outside of its translation unit.

(staticFunction)